public/context.json

```diff
@@ -35,7 +36,10 @@
     "title": {"@id": "dcterms:title"},
     "abstract": {"@id": "dcterms:abstract"},
 
+
     "edition": {"@id": "bf:editionStatement"},
+    "projection": {"@id": "bf:cartographicProjection"},
+    "scale": {"@id": "bf:cartographicScale"},
```

Comment:
Alas, projection and scale both look like object properties that point to classes instead of literals:

http://id.loc.gov/ontologies/bibframe.html#c_Projection
http://id.loc.gov/ontologies/bibframe.html#c_Scale